ci-operator/jobs/openshift/build-test-images/openshift-build-test-images-master-presubmits.yaml

```diff
@@ -0,0 +1,48 @@
+presubmits:
+  openshift/build-test-images:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: build01
+    context: ci/prow/images
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-build-test-images-master-images
+    rerun_command: /test images
+    spec:
+      containers:
+      - args:
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --report-password-file=/etc/report/password.txt
+        - --report-username=ci
+        - --target=[images]
+        - --target=[release:latest]
```

Comment:
@stevekuznetsov adding a `promotion` stanza appears to result in this target being added to the presubmit job.  And this target then fails if you do not have a `tag_specification`

Reply:
I did rerun `make update` after removing the `tag_specification` but leaving the `promotion`

Reply:
yeah, the promotion stanza is what leads to this --target arg being added.  the tag_specification is probably what provides the target (under the covers somewhere).

i think i would have expected tag_specification to be what added this --target arg.